src/main/java/com/iesam/digitallibrary/features/digitalresource/audiobook/presentation/AudiobookPresentation.java

```diff
@@ -55,4 +65,25 @@ public static void createAudiobook(){
         newAudiobookUseCase.execute(newAudiobook);
     }
 
+    public static void getAudiobooks(){
+        ListAudiobooksUseCase listAudiobooksUseCase = new ListAudiobooksUseCase(new AudiobookDataRepository(new AudiobookFileLocalDataSource()));
+        List<Audiobook> audiobooks = listAudiobooksUseCase.execute();
+        System.out.println(audiobooks);
+    }
+
+    public static Audiobook getAudiobook(){
+        System.out.println("ISBN of Audiobook to list: ");
+        scanner.nextLine();
+        String isbn = scanner.nextLine();
+
+        GetAudiobookUseCase getAudiobookUseCase = new GetAudiobookUseCase(new AudiobookDataRepository(new AudiobookFileLocalDataSource()));
+        Audiobook audiobook = getAudiobookUseCase.execute(isbn);
+
+        if (audiobook != null) {
```

Comment:
El uso del condicional "if" para verificar si un audiolibro es nulo no está correctamente planteado, ya que si el audiolibro es nulo, no tendría una ID; debe ser nulo en su totalidad para que esta condición sea cierta. En su lugar, lo que haría es comprobar si esa ID existe, o en su defecto, que no haya ningún valor. Dependiendo de cómo se quiera plantear la lógica, si se desea que solo se introduzca una ID hasta que corresponda a un audiolibro válido, se debería aplicar esa verificación específica; y si se prefiere que no se permita introducir nada hasta tener una ID válida, entonces se debe implementar la lógica opuesta.
```suggestion
        if (audiobook != null) {
```suggestion

Reply:
Vale, realizaré un cambio para que en el caso de uso se compruebe si el ISBN del Audiolibro existe. Si existe se listará sin problema pero, si no existe, entonces lanzará una excepción. Después esto se recogerá en un bloque try-catch en la capa de presentación.